content/en/funding/meetup-funding.mdx

```diff
@@ -180,15 +192,15 @@ How and where to promote your meetup.
 
 ### During the Event
 
-If we fund your meetup we’ll ask you to speak briefly about Community Development at MakerDAO.
+If we fund your meetup, we’ll ask you to speak briefly about Community Development at MakerDAO.
 
 <Checklist>
 
-Direct people to the [Community Portal]()
+Direct new users to the [Community Portal.]()
 
-Talk about the different ways to contribute to MakerDAO
+Outline the different ways to contribute to MakerDAO.
 
-Mention that we award Dai bounties varying by the type of contribution.
+Mention that we Dai bounty awards vary by type of contribution.
```

Comment:
Mention that Dai bounty awards vary by type of contribution.

accidentally a word